.env.example

```
@@ -12,4 +12,4 @@ MONGODB_URI=mongodb://localhost:27017/monew
 AWS_ACCESS_KEY=your_access_key
 AWS_SECRET_KEY=your_secret_key
 AWS_REGION=ap-northeast-2
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_KEY key

(UnorderedKey)